voluncheer/opportunityboard/views/search.py

```diff
@@ -65,10 +66,32 @@ def search(self):
             filtered_opportunity = filtered_opportunity.filter(subcategory=self.subcategory)
         if self.subsubcategory is not None:
             filtered_opportunity = filtered_opportunity.filter(subsubcategory=self.subsubcategory)
-
+        if self.duration == "0-2 Hours":
+            filtered_opportunity = filter_by_duration(filtered_opportunity, 0, 2)
+        if self.duration == "2-4 Hours":
+            filtered_opportunity = filter_by_duration(filtered_opportunity, 2, 4)
+        if self.duration == "4-8 Hours":
+            filtered_opportunity = filter_by_duration(filtered_opportunity, 4, 8)
+        if self.duration == "8+ Hours":
+            filtered_opportunity = filter_by_duration(filtered_opportunity, 8, 24)
```

Comment:
If this maxes out at 24 hours it's not really 8+ but rather 8-24, right?

Reply:
Due to the changes above (searching for durations less than equal to max) this is no longer an issue. Not sure if we will want allow for opportunities greater than 8 hours and how we should search for those, but will discuss with Hanna and address if needed.

Reply:
Originally, I tried to think about typical time slots people would consider when signing up for an opportunity, and I first thought of a small, part-time, and full-time opportunity. I assumed that an opportunity would not typically exceed 8 hours.

Reply:
For now, I think that we can assume an opportunity 6 hours or above is considered full time, and would therefore show up in a search for full time opportunities